package.json

```diff
@@ -43,7 +43,14 @@
     "parser": "babel-eslint",
     "rules": {
       "global-require": 0,
-      "import/no-unresolved": [2, {"ignore": ["atom"]}]
+      "import/no-unresolved": [
```

Comment:
`npm --save(-dev)` apparently restructures the style of the rest of the file :confused:.